README.md

```diff
@@ -68,6 +68,12 @@ Poetry run pytest
 ```
 (please make usre you run 'poetry install' beforehand to install 'pytest')
 
+If intead you would like to run your tests via Docker, please run the followig:
```

Comment:
Couple of typos here - I know the docs don't fuss as much as the code when you're out by a character, but we should try to minimise these as they can make it harder to read for future devs!
* `instead`
* `following`